pieman/bin/compress.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python3
+# Copyright (C) 2019 Denis Gavrilyuk <karpa4o4@gmail.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+"""
+
+import argparse
+import bz2
+import gzip
+import os
+from lzma import FORMAT_AUTO, LZMADecompressor, LZMAError, \
+compress as compress_lzma
+
+
+def decompress_lzma(data, format=FORMAT_AUTO, memlimit=None, filters=None):
```

Comment:
Shadowing a built-in (`format` in this case) is always a bad idea. Choose a different name.